cerberus/client.py

```diff
@@ -55,6 +56,104 @@ def get_token(self):
         """Returns a client token from Cerberus"""
         return self.token
 
+    def get_roles(self):
+        """Returns all the roles (IAM or User Groups) that can be granted to a safe deposit box.
+
+        Roles are permission levels that are granted to IAM or User Groups.  Associating the id for the write role
+          would allow that IAM or User Group to write in the safety deposit box."""
+        sdb_resp = requests.get(self.cerberus_url + '/v1/role',
+                                headers=self.HEADERS)
+
+        self.throw_if_bad_response(sdb_resp)
+        return sdb_resp.json()
+
+    def get_role(self, key):
+        """Returns id of named role."""
+
+        sdb_resp = requests.get(self.cerberus_url + '/v1/role',
```

Comment:
Same

Reply:
This could probably just call the `get_roles` method?

Reply:
You're right!  I'll change that around.